CMSgov/design-system
# [WNMGDS-2750] Remove `aria-disabled` from `TabPanel` components

## Summary

- Removes deprecated `aria-disabled` attribute from `TabPanel` components to align with current W3C guidance, which advises against using `aria-disabled` on elements with `role="tabpanel"`.
- Updates a unit test to ensure that disabled `TabPanel` components are marked with `aria-hidden="true"`.

See the [Jira story and commentary](https://jira.cms.gov/browse/WNMGDS-2750) for context.

## How to test

- Open the Tabs component's disabled story in Storybook.
- Use a screen reader (e.g., VoiceOver) to navigate the tabs.
- Confirm that the screen reader announces disabled tabs as "dimmed," indicating they are non-interactive.

## Checklist
